packages/coding-standard/docs/phpstan_rules.md

```diff
@@ -457,6 +457,30 @@ final class SomeClass
 
 <br>
 
+## No Trait Except For Symfony Class
+
+- class: [`NoTraitExceptForSymfonyClassRule`](../src/Rules/NoTraitExceptForSymfonyClassRule.php)
+- **configuration allowed**
+
+```yaml
+# phpstan.neon
+rules:
+    - Symplify\CodingStandard\Rules\NoTraitExceptForSymfonyClassRule
+```
```

Comment:
If there is some configuration allowed, it should be part of the Yaml example, as in other rule here